testsuite/python/widom_insertion.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright (C) 2013-2018 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""Testmodule for the Widom Insertion.
+"""
+import os
+import sys
+import unittest as ut
+import numpy as np
+import espressomd  # pylint: disable=import-error
+from espressomd import reaction_ensemble
+from tests_common import lj_potential
+
+
+@ut.skipIf(not espressomd.has_features(["LENNARD_JONES"]),
+           "Features not available, skipping test!")
+class ReactionEnsembleTest(ut.TestCase):
+
+    """Test the implementation of the widom insertion (part of the reaction ensemble)."""
+
+    # The excess chemical potential is calculated for identical particles into
+    # a 20 cubed box with a single particle, interacting via a LJ-potential
+    # (cut-off at 5 sigma).
+    N0 = 1
+    temperature = 0.5
+    type_HA = 0
```

Comment:
Constants are normally written all-capital in python according to [pep8](https://www.python.org/dev/peps/pep-0008/#id48)